### PR TITLE
Style fixes for new management reports

### PR DESCRIPTION
**Pull request by nop33**

### CODE REVIEW

File: indico/htdocs/js/indico/modules/sessions.js [hunk 1]
```diff
@@ -46,7 +46,9 @@
             setupTableSorter();
             setupPalettePickers();
         }).on('click', '.show-session-blocks', function() {
-            $(this).closest('tr').toggleClass('selected').nextUntil('tr:not(.session-blocks-row)', 'tr').toggle();
+            if($(this).data('count')) {
```
Comment (ThiefMaster) [inline indico/htdocs/js/indico/modules/sessions.js]: missing space after `if`

File: indico/htdocs/js/indico/modules/sessions.js [hunk 2]
```diff
@@ -46,7 +46,9 @@
...
+                $(this).closest('tr').toggleClass('selected').nextUntil('tr:not(.session-blocks-row)', 'tr').toggle();
+            };
```
Comment (ThiefMaster) [inline indico/htdocs/js/indico/modules/sessions.js]: no semicolon after a block

File: indico/htdocs/js/indico/modules/sessions.js (hunk 1 above)
Comment (ThiefMaster) [inline indico/htdocs/js/indico/modules/sessions.js]: var $this = $(this);

File: indico/htdocs/js/indico/modules/sessions.js (hunk 2 above)
Comment (nop33) [inline indico/htdocs/js/indico/modules/sessions.js]: Damn Sublime :tongue:

File: indico/htdocs/js/indico/modules/sessions.js [hunk 3]
```diff
@@ -46,7 +46,10 @@
             setupTableSorter();
             setupPalettePickers();
         }).on('click', '.show-session-blocks', function() {
-            $(this).closest('tr').toggleClass('selected').nextUntil('tr:not(.session-blocks-row)', 'tr').toggle();
+            var $this = $(this);
+            if ($this.data('count')) {
+                $this.closest('tr').toggleClass('selected').nextUntil('tr:not(.session-blocks-row)', 'tr').toggle();
+            };
```
Comment (ThiefMaster) [inline indico/htdocs/js/indico/modules/sessions.js]: :monkey:

**Pull request merged**